### PR TITLE
Implement Admin\Orders::maybe_remove_order_metaboxes()

### PR DESCRIPTION
**Pull request by unfulvio**
# Summary

This PR implements the `maybe_remove_order_metaboxes()`

## UI Changes

The order actions meta box is removed if you have an order created with FB Commerce that is pending payment.

## Open questions

I have encountered an issue while implementing an acceptance test for this and could use a little help.

The user testing seems consistently removing the meta box. If I try to create an order in acceptance suite that is pending and created with `facebook` or `instagram`, the meta box persists.

I am not sure why. Could it be a flaky test failing on my instance only? Have I missed any step in setting up the order? The rest of the tests pass...

#### User testing

1. Go to WooCommerce > Orders
1. Click on a Order 
    - [x] The "Order actions" meta box is visible
1. Update the order via `set_created_via()` using `facebook` or `instagram`
1. Set the order status to `pending`, save and reload the order page
    - [x] The "Order actions" meta box is no longer output
1. Change the order status to `complete`, `processing` or other status than pending
    - [x] The meta box is back

#### Acceptance testing

- [x] `acceptance/Admin/OrderRefundsCest.php` passes (see my note above)

## Before merge

- [x] I have confirmed these changes in each supported minor WooCommerce version

### CODE REVIEW

File: includes/Admin/Orders.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function add_hooks() {
 		add_action( 'woocommerce_email_enabled_customer_processing_order', [ $this, 'maybe_stop_order_email' ], 10, 2 );
 		add_action( 'woocommerce_email_enabled_customer_refunded_order', [ $this, 'maybe_stop_order_email' ], 10, 2 );
 
-		add_action( 'admin_menu', [ $this, 'maybe_remove_order_metaboxes' ] );
```
Comment (unfulvio) [inline includes/Admin/Orders.php:60]: this hook wasn't working for me so I changed it

File: tests/_support/AcceptanceTester.php [hunk 2]
```diff
@@ -158,6 +158,20 @@ public function haveVariableProductInDatabase( array $args = [] ) {
 	}
 
 
+	/**
+	 * Creates an order in the database.
+	 *
+	 * @return \WC_Order
+	 */
+	public function haveOrderInDatabase() {
```
Comment (unfulvio) [inline tests/_support/AcceptanceTester.php:166]: can be expanded like the product equivalent if needed in future

**Comment by N-Molham** [comment]
@unfulvio I am trying to runt the test but it gives me this error 
```
1) OrderRefundsCest: Try_maybe_remove_order_metaboxes
 Test  tests\acceptance\Admin\OrderRefundsCest.php:try_maybe_remove_order_metaboxes

  [ConnectionException] Can't connect to Webdriver at http://127.0.0.1:4444/wd/hub. Please make sure that Selenium Server or PhantomJS is running.

#1  D:\xampp\htdocs\skyverge-facebook-for-woocommerce\vendor\symfony\event-dispatcher\EventDispatcher.php:264
#2  D:\xampp\htdocs\skyverge-facebook-for-woocommerce\vendor\symfony\event-dispatcher\EventDispatcher.php:239
#3  D:\xampp\htdocs\skyverge-facebook-for-woocommerce\vendor\symfony\event-dispatcher\EventDispatcher.php:73
```
I guess you need either Selenium or PhantonJS running ... do you have a reference to the setup and configuration?

**Comment by unfulvio** [comment]
hey @N-Molham you don't need PhantomJS, as Chrome will do; but you do need a Selenium server running along while Codeception does its thing; setting it up may be platform dependent (something we wanted to get around with our Lumiere project using Docker but FB doesn't use it). Here you can find a guide for Memberships about Selenium and Codeception/WPBrowser: https://github.com/skyverge/woocommerce-memberships/wiki/Selenium - hopefully you could just use the npm package and it's more straightforward; if neither solution there works, try googling how to run Selenium from the WSL; then, while that process runs, try running codeception.

**Review by N-Molham — APPROVED** [review]
Looking good

**Pull request merged**